scriptlex.py

```diff
@@ -56,7 +56,7 @@ def t_ID(t):
     return t
     
 def t_COMMAND(t):
-    r'(?m)^\s*/.+'
+    r'(?m:^\s*\/.+)'
```

Comment:
This line literally does nothing anyways if i see this correctly, since it's just a freestanding String literal. Why are we changing it then (we should probably be either removing it or leaving it alone)

Reply:
Why do you say it does nothing? It matches specific lines that are interpreted as literal Minecraft commands.